tubemq-core/src/main/java/org/apache/tubemq/corebase/configuration/ConfigOption.java

```diff
@@ -0,0 +1,302 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.tubemq.corebase.configuration;
```

Comment:
tubemq-core/src/main/java/org/apache/tubemq/corebase/configuration/*

I found that part of the content in this directory is directly copied from flink.

If you want to use it directly, you need to declare its corresponding license and notice in LICENSE and NOTICE, such as https://github.com/apache/incubator-inlong/ Processing of section 1.3.1 in blob/master/LICENSE#L329 and https://github.com/apache/incubator-inlong/blob/master/NOTICE#L9

By the way, can you digest this piece and implement it with your own ideas? You can also see that the LICENSE and NOTICE of most open source projects are very concise, indicating that most of them are written by themselves.

It doesn't matter if you haven't considered it for the time being, we can improve it later.

Reply:
@gosonzhang  Thank you for your comments. IMO,  we should learn from good design while keeping the LICENSE and NOTICE concise. So I only introduced a small part of auxiliary classes. If necessary, I will optimize again to reduce the size of these auxiliary classes.

Reply:
This should be a trend, look at Kafka, Flink, Pulsar, etc., except for a few core codes of them are directly quoted, most of them are self-developed or deeply modified.

There are many implementation options for the configuration part, if it expands the LICENSE and NOTICE of the main project, then those who analyze and use our project should think that this project is a hodgepodge

Reply:
When TubeMQ was first open sourced, a lot of external component code was cited in the project, which was criticized by the outside. Now we are developing new features while replacing and cleaning the referenced external code. In the end, we will only keep a small amount of external code references.

Reply:
I now probably understand the main meaning of your guidance. And I made some changes based on your reply. Please take a look. Thank you for your elaboration and guidance.